WcaOnRails/app/models/result.rb

```diff
@@ -80,16 +80,47 @@ def average_is_not_computable_reason
     event ? invalid_solve_count_reason : "Event needed to compute average"
   end
 
+  def should_compute_average?
+    # Average of 5 and Mean of 3 rounds should definitely attempt to compute the average (the average
+    # may still be empty because of combined rounds).
+    # Best of 3 is weird. We actually do want to populate the average column for best of 3 with:
+    #  - 333fm was changed from allowing best of 3 (and disallowing mean of 3) to allowing mean of 3 (and disallowing best of 3).
+    #    See "Relevant regulations changes" below.
+    #    With this change in format, the Board decided to compute means for all
+    #    the old best of 3 rounds, and assign mean records to them.
+    #    However, we could not change the format of the rounds, as that would have affected rankings
+    #    in past competitions, so the rounds remain as best of 3, but with an average computed.
+    #  - 333ft has a similar story to 333fm. It also changed from allowing best of 3
+    #    (and disallowing mean of 3) to allowing mean of 3 (and disallowing best
+    #    of 3). See "Relevant regulations changes" below.
+    #    this event, but fortunately, no competition ever ran 333ft as an average of 5.
```

Comment:
Ah, whoops! That was a typo (leftover sentence I did not fully delete). Good catch =)